### ISSUE TITLE
Labels skewed when viewing after submission

### ISSUE THREAD

**Issue by jerowe** [heartexlabs/label-studio#276]
**Describe the bug**
When I first go through and label an image using brush the labels appear as they should. Then, after I submit if I go back to 'Tasks' and view the image the labels are skewed. I think the labels are still correct, just displayed funny.

**To Reproduce**
![1649_1109_0003_Amp5-1_B_20070424_A01_w2_15ADF48D-C09E-47DE-B763-5BC479534681-autolevel](https://user-images.githubusercontent.com/5555526/80949624-03e80100-8dfd-11ea-9ff0-cb7f65345812.png)

Here's the config - 

```
<View>
  <BrushLabels name="tag" toName="img">
    <Label value="Alive" background="green" />
    <Label value="Dead" background="red" />
  </BrushLabels>
  <Image name="img" value="$image" zoom="true" zoomControl="true" />
</View>
```
1. Go to tasks
2. Label the image
3. Submit
3. Go back to Tasks
4. View the submitted task
5. Labels are skewed.

**Expected behavior**

**Screenshots**
During labeling
<img width="1501" alt="Screenshot 2020-05-04 11 44 21" src="https://user-images.githubusercontent.com/5555526/80949787-5b866c80-8dfd-11ea-90a1-df5cf74c1027.png">
Viewing from the Tasks screen 
<img width="1531" alt="Screenshot 2020-05-04 11 45 04" src="https://user-images.githubusercontent.com/5555526/80949809-64773e00-8dfd-11ea-9adf-446b58e3aa92.png">

**Environment (please complete the following information):**
 - OS: Running with docker [(latest image)](https://hub.docker.com/layers/heartexlabs/label-studio/latest/images/sha256-89ae4d79f46bdd825958165062fb7acbe0ef9bf8f5965b0a6abaa8f5eab00a1f?context=explore)
 - Browser Chrome

**Additional context**
Not sure if it's helpful or not, but I get this error from the console. 

```
Konva warning: The stage has 6 layers. Recommended maximum number of layers is 3-5. Adding more layers into the stage may drop the performance. Rethink your tree structure, you can use Konva.Group.
```

**Comment by makseq** [comment]
@jerowe We've released a new LS version 0.7.0. Can you check is this bug persistent in the new version?

**Comment by mcombalia** [comment]
Hello,

I'm in version 0.7.3 and I'm still seeing inconsistencies with the segmentation with brushes. The segmentation gets displaced and scaled when I save it (or update it) and query it again.

Thanks!

**Comment by makseq** [comment]
Yes, we are working on it. Hope we will fix it in the nearest release. 

Related issue: https://github.com/heartexlabs/label-studio/issues/374

**Comment by mcombalia** [comment]
Great thanks. Is it only a visualization problem, or the labels are not correctly saved? This is to understand if I can still use the tool to label regions.

**Comment by makseq** [comment]
@hlomzik I think you know it better.

**Comment by makseq** [comment]
@jerowe @mcombalia We've just released label-studio 0.7.4rc2 with a lot of fixes regarding to brushes. The issue with skew and resizes must be fixed there. If it's still actual, please check it. 

```
pip install label-studio==0.7.4rc2
```

**Comment by makseq** [comment]
Fixed in https://github.com/heartexlabs/label-studio/releases/tag/v0.7.4.
